server/seeds/demoSeed.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+const User = require("../models/User");
+const connectDB = require("../db");
+
+connectDB();
+
+const seedUser = [
+    {
+        name: 'demouser123',
+        email: 'demouser123@gmail.com',
+        password: 'demouserpassword123',
+    },
+];
+
+const seedDB = async () => {
+    await User.insertMany(seedUser);
```

Comment:
We also need to make the profile too 🙂

Reply:
I've updated it to create a profile as well.